package.json

```diff
@@ -1,28 +1,63 @@
 {
-  "name": "fiscal",
-  "version": "1.0.0",
+  "name": "@winstoncl/fiscal",
+  "version": "2.1.0",
   "description": "Computes fiscal quarter and year date ranges",
-  "main": "index.js",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
   "repository": {
     "type": "git",
-    "url": "https://github.com/MarkHerhold/fiscal.git"
+    "url": "https://github.com/winstoncl/fiscal.git",
+    "directory": "@winstoncl/fiscal"
+  },
+  "publishConfig": {
+    "@winstoncl:registry": "https://npm.pkg.github.com"
   },
   "scripts": {
-    "test": "eslint . --ignore-pattern '*.min.js' && mocha test/",
-    "dist": "browserify index.js --standalone Fiscal -t [ babelify --presets [ es2015 ] ] -o fiscal.min.js",
-    "minify": "uglifyjs fiscal.min.js --screw-ie8 --preamble \"/* Fiscal by Mark Herhold - MIT Licensed */\" -o fiscal.min.js"
+    "test": "TZ=UTC jest --config jestconfig.json",
+    "format": "prettier --write \"src/**/*.ts\" \"src/**/*.js\"",
+    "lint": "tslint -p tsconfig.json",
+    "build": "tsc",
+    "dist": "browserify src/index.ts --standalone Fiscal -t [ babelify --presets [ es2015 ] ] -o fiscal.min.js -p [ tsify --noImplicitAny ]",
+    "minify": "terser --compress --mangle --format 'preamble=\"/* Fiscal by Mark Herhold - MIT Licensed */\"' -o fiscal.min.js -- fiscal.min.js "
   },
+  "files": [
+    "dist"
+  ],
   "keywords": [
     "fiscal",
     "quarter"
   ],
-  "author": "Mark Herhold <mherhold@rti.org>",
+  "contributors": [
+    {
+      "name": "Mark Herhold",
+      "email": "mherhold@rti.org",
+      "url": "https://github.com/MarkHerhold"
+    },
+    {
+      "name": "Winston Poh",
+      "email": "winston@commonledger.com",
+      "url": "https://github.com/winstoncl"
+    }
+  ],
   "license": "MIT",
   "devDependencies": {
+    "@types/jest": "^27.4.0",
+    "@types/node": "^17.0.18",
     "babel-preset-es2015": "^6.3.13",
     "babelify": "^7.2.0",
     "chai": "^3.4.1",
     "chai-datetime": "^1.4.0",
-    "mocha": "^2.3.4"
+    "jest": "^27.5.1",
+    "mocha": "^2.3.4",
+    "prettier": "^2.5.1",
+    "ts-jest": "^27.1.3",
+    "tslint": "^6.1.3",
```

Comment:
tslint is actually deprecated now 
https://palantir.github.io/tslint/
https://www.darraghoriordan.com/2020/03/06/upgrade-tslint-deprecated-to-eslint/